scripts/create_tables.sql

```diff
@@ -1 +1,30 @@
 -- create doctors, patients, and appointments tables
+  CREATE TABLE doctors (
+    id SERIAL PRIMARY KEY,
+    family_name TEXT,
+    given_name TEXT,
+    gender TEXT,
+    born_on DATE,
+    specialty TEXT
+  );
+
+  CREATE TABLE patients(
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  height INTEGER,
+  weight INTEGER,
+  born_on DATE,
+  insurance TEXT
+);
+
+  CREATE TABLE appointments (
```

Comment:
This table is missing:
- `date` column of type `DATE`.
- `time` column of type `TIME`.
- `comment` column of type `TEXT`.

```sql
CREATE TABLE appointments(
  id SERIAL PRIMARY KEY,
  date DATE,
  time TIME,
  comment TEXT,
  doctor_id INTEGER REFERENCES doctors(id),
  patient_id INTEGER REFERENCES patients(id)
);
```